something.css

```diff
@@ -0,0 +1,3 @@
+.hook {
+  mask: url('');
```

Comment:
CSS Masks not supported by: IE (8,9,10,11), Firefox (37,38), Chrome (42,43), Safari (8), iOS Safari (7.0-7.1,8.1-8.3), Opera Mini (5.0-8.0), Android Browser (4.2-4.3,4.4,4.4.3-4.4.4), Chrome for Android (42), UC Browser for Android (9.9)